src/components/MarkdownProvider/components/Typography.tsx

```diff
@@ -66,7 +69,7 @@ export const StyledPre: React.FC = ({ children }) => {
 };
 
 export const StyledStrong = styled.strong`
```

Comment:
Fwiw, eventually we'll map this to `styled(Span).attrs({ tag: 'strong', isBold: true })` (when `Span` exists)